pmemcheck/pmc_main.c

```diff
@@ -1753,41 +1742,6 @@ pmc_handle_client_request(ThreadId tid, UWord *arg, UWord *ret )
             break;
         }
 
-        case VG_USERREQ__PMC_FULL_REORDED: {
```

Comment:
Please add switch for reserved values with an error message, just in case of someone messing up his environment.

Reply:
Done.